### ISSUE TITLE
Title bar changes color(from black to white) when I scroll down then scroll up

### ISSUE THREAD

**Issue by rmakki** [yazeed44/MultiImagePicker#77]
When I scroll down through the Albums, the title bar(black background/white text) gets hidden but then when I scroll up, the title bar appears again with a white background and no text.  Not sure if this is a bug or it has to do with my styles. Below is a copy of my manifest and my gradle dependencies. Do you have a sample code using your library that I can look at, it helps to see what basic configurations you had used to get your code working.

I ran this on Samsung Galaxy S6 edge.

<?xml version="1.0" encoding="utf-8"?>
<resources>
    
    <style name="AppTheme" parent="android:Theme.Holo.Light.DarkActionBar">
        
    </style>
    <style name="MIP_theme" parent="Theme.AppCompat.Light.DarkActionBar">
        <item name="windowActionBar">false</item>
        <item name="windowNoTitle">true</item>
    </style>
</resources>

Here are the dependencies from my gradle file

dependencies {

    compile fileTree(dir: 'libs', include: ['*.jar'])
    compile 'com.android.support:appcompat-v7:23.0.1'
    compile 'com.android.support:design:22.2.0'
    compile 'net.yazeed44.imagepicker:imagepicker:1.3.0'

}

**Comment by yazeed44** [comment]
Yes i had this problem too, unfortunely i don't know the exact problem but maybe it's a bug in support libraries.
I'm really busy these day with college, so i can't debug it :(

**Comment by rmakki** [comment]
Thanks Yazeed, do you have a sample code using your library to share?

**Comment by yazeed44** [comment]
Yes. It's in app directory

https://github.com/yazeed44/MultiImagePicker/blob/master/app/src/main/java/net/yazeed44/multiimagepicker/MainActivity.java
